orchestrate.py

```diff
@@ -69,13 +69,14 @@ def get(self, path=None):
         self.render("loading.html", path=path)
 
 
-class StatsHandler(BaseHandler):
+class APIStatsHandler(BaseHandler):
     def get(self):
```

Comment:
While you're at it, can you add `self.set_header("Content-Type", 'application/json')` to the APIHandler?